internal/server/iigointernal/judiciary.go

```diff
@@ -410,3 +528,23 @@ func searchEvalReturnForRuleName(name string, evaluationReturn roles.EvaluationR
 	}
 	return false
 }
+
+// mergeEvalResults takes two evaluation results and merges them to provide a unified set
+func mergeEvalResults(set1 map[shared.ClientID]roles.EvaluationReturn, set2 map[shared.ClientID]roles.EvaluationReturn) map[shared.ClientID]roles.EvaluationReturn {
+	for key, val := range set1 {
+		if set2Val, ok := set2[key]; ok {
+			defRules := set2Val.Rules
```

Comment:
why not write a `mergeEvaluationReturn` function?

Reply:
Yeah that function is poorly named. Renaming to mergeEvaluationReturn, since that's essentially what it does.

Reply:
Ahh,I meant a function that just deals with two `roles.EvaluationReturn`. But if you tested this and it's fine, I'm OK with this.